README.md

```diff
@@ -15,7 +15,17 @@ Or install:
 
 Configure it in an initializer:
 
+    Sidekiq.configure_client do |config|
+      config.client_middleware do |chain|
+        chain.add Sidekiq::Middleware::Client::Datadog
+      end
+    end
+
     Sidekiq.configure_server do |config|
+      config.client_middleware do |chain|
+        chain.add Sidekiq::Middleware::Client::Datadog
+      end
```

Comment:
`Sidekiq::Middleware::Client::Datadog` is needed in `Sidekiq.configure_server` ?

Reply:
ok. thanks 😄